kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/DisbursementVoucherNonEmployeeTravel.xml

```diff
@@ -0,0 +1,43 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans 
+		http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+	<bean id="DisbursementVoucherNonEmployeeTravel-disbVchrServicePerformedDesc-parentBean" abstract="true" parent="AttributeDefinition">
+		<property name="name" value="disbVchrServicePerformedDesc" />
+		<property name="forceUppercase" value="false" />
+		<property name="label" value="Business Purpose" />
+		<property name="shortLabel" value="Purpose" />
+		<property name="maxLength" value="50" />
+		<property name="required" value="true" />
+		<property name="control">
+			<bean parent="TextControlDefinition" p:size="50" />
+		</property>
+	</bean>
+
+	<bean id="DisbursementVoucherNonEmployeeTravel-dvServicePerformedLocName-parentBean" abstract="true" parent="AttributeDefinition">
+		<property name="name" value="dvServicePerformedLocName" />
+		<property name="forceUppercase" value="false" />
+		<property name="label" value="Destination" />
+		<property name="shortLabel" value="Place" />
+		<property name="maxLength" value="40" />
+		<property name="required" value="true" />
+		<property name="control">
+			<bean parent="TextControlDefinition" p:size="40" />
+		</property>
+	</bean>
+
+	<bean id="DisbursementVoucherNonEmployeeTravel-dvServiceRegularEmprName-parentBean" abstract="true" parent="AttributeDefinition">
```

Comment:
We should override the child bean instead of the parent bean and update the label and shortlabel fields. The other fields should pull from the parent bean. Look at edu.arizona.kfs.fp.businessobject.datadictionary.ProcurementCardHolder.xml.